modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,63 @@
+
+* The Knative TLS authentication feature for internal traffic is now depricated. The functionality with the `internal-encryption` configuration flag is no longer available.
+
+* The secret filtering is enabled by default on the {ServerlessOperatorName} side. An environment variable, `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=true`, is added by default to the `net-istio` and `net-kourier` controller pods.
+
+* The `domain-mapping` and `domain-mapping-webhook` deployments functionality in the `knative-serving` namespace is now removed.
+
+* If you have set `spec.config.domain` in the KnativeServing Custom Resource (CR), the default external domain will no longer auto-populate the config-domain ConfigMap in the `knative-serving` namespace. You will now need to configure it manually to ensure accurate domain settings. 
+
+* You can now use the gRPC health probe for `net-kourier` deployments. The `net-kourier-controller` now uses a standard Kubernetes gRPC health probe for both readiness and liveness, replacing it's previous use of exec and custom commands. This simplifies the process and aligns it with Kubernetes standards. Additionally, the timeoutSeconds value has been adjusted from 100 milliseconds to 1 second to ensure more reliable probe responses.
+
+* The new trigger filters feature is now available as a Technology Preview. The new trigger filters are now enabled by default. It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
+
+* Knative Eventing now offers support for data in transit encryption (Eventing TLS) as a developer preview. You can configure Knative Eventing components to encrypt data in-transit by exposing HTTPS addresses and accepting user-provided CA trust bundles. 
+
+* You can now use the Custom Metrics Autoscaler (KEDA) operator to autoscale Knative Eventing sources for Apache Kafka (KafkaSource), both up and down. This functionality is available as a developer preview, offering enhanced scalability and efficiency for Kafka-based event sources within Knative Eventing.
+
+* You can now explore the Knative Eventing monitoring dashboards directly within the *Observe* tab of the developer view in the OpenShift Developer Console. 
+
+* The support for EventTypes `v1beta1` in Knative shipped is deprecated in {ServerlessProductName} 1.32. The new Knative CLI does not function with previous versions of Serverless, as it is limited only to the EventTypes resource, significantly narrowing its functionality. In {ServerlessProductName} 1.32, the Knative CLI utilizes the EventType v1beta2 API to facilitate the new reference model. The `kn` is not backward compatible with the EventType API v1beta1 in previous releases, specifically limited to the `kn eventtypes` sub-commands group. Therefore, it is recommended to use a matching `kn` version for the best user experience.  
```

Comment:
```suggestion
* The support for EventTypes `v1beta1` in Knative shipped is deprecated in {ServerlessProductName} 1.32. In {ServerlessProductName} 1.32, the Knative CLI utilizes the EventType v1beta2 API to facilitate the new reference model. The `kn` is not backward compatible with the EventType API v1beta1 in previous releases, specifically limited to the `kn eventtypes` sub-commands group. Therefore, it is recommended to use a matching `kn` version for the best user experience.  
```
@kaldesai 

I'd drop this sentence. I feel like it might create a bit of confusion for unfamiliar reader. It's better explained and reiterated in the following ones.